omero/sysadmins/version-requirements.txt

```diff
@@ -502,27 +622,27 @@ OMERO support policies
     * - 3.5
       - from Mar 2013
       - to Oct 2013
-      - Recommended
-      - Deprecated
-      - Dropped
+      - |Recommended|
+      - |Deprecated|
+      - |Dropped|
       - :zerocforum:`Ice 3.5 <announcements/6093-ice-3-5-0-released>`
         :zerocforum:`Ice 3.5.1 <announcements/6283-ice-3-5-1-released>`
     * - 3.6
       - from June 2015
       - to TBA
-      - Supported
-      - Recommended
-      - Supported
+      - |Supported|
+      - |Recommended|
+      - |Supported|
       - :zerocforum:`Ice 3.6.0 <announcements/6631-ice-3-6-0-and-ice-touch-3-6-0-released>`
-        :zerocforum:`Ice 3.6.1 <announcements/45941-ice-3-6-0-and-ice-touch-3-6-1-released>` (broken)
+        :zerocforum:`Ice 3.6.1 <announcements/45941-ice-3-6-0-and-ice-touch-3-6-1-released>` |Broken|
```

Comment:
Another case where the lack of being able to do line-breaks makes it difficult - I could put this version and the symbol in brackets, would that help?